man/man1/kbsecret-cp.1.ronnpp

```diff
@@ -0,0 +1,26 @@
+kbsecret-cp(1) - copy kbsecret(1) record(s)
+===========================================
+
+## SYNOPSIS
+
+`kbsecret cp` <src_sess> <dst_sess> [options] <record ![record ...]>
+
+## DESCRIPTION
+
+`kbsecret cp` copies the requested record(s) between sessions
```

Comment:
"from one session to another, optionally deleting the originally afterwards" to be a little more explicit 🙂